examples/rerun/run_tests.rs

```diff
@@ -0,0 +1,61 @@
+//! This is an example lang_tester that shows how to use the "rerun_if" commands.
+
+use std::{
+    env,
+    fs::{read_to_string, remove_file},
+    path::PathBuf,
+    process::Command,
+};
+
+/// In order that we can meaningfully have the same test behave differently from run to run, we
+/// have to leave cookies behind in target/tmp, which we must clear before starting a test run.
+/// This is horrible, but without doing something stateful, we can't test anything.
+const COOKIES: &[&str] = &[
+    "rerun_status_cookie",
+    "rerun_stderr_cookie",
+    "rerun_stdout_cookie",
+];
+
+use lang_tester::LangTester;
+use regex::Regex;
+
+fn main() {
+    for cookie in COOKIES {
+        let mut p = PathBuf::new();
+        p.push(env::var("CARGO_MANIFEST_DIR").unwrap());
```

Comment:
It doesn't hugely matter, but do you know the `[elems, ...].collect::<PathBuf>()` trick?

Reply:
I didn't!

Reply:
It's not always practical, but often makes building paths a one-liner.

Reply:
Just tried this, and it turns 5 lines into 8 lines (since `iter` is also needed) so not a win this case it seems. Oh well, it's a useful trick to know!

Reply:
Oh well.